frames/src/main/java/com/checkout/frames/mapper/CardSchemeComponentStyleToViewStyleMapper.kt

```diff
@@ -15,7 +15,10 @@ internal class CardSchemeComponentStyleToViewStyleMapper(
 ) : Mapper<CardSchemeComponentStyle, CardSchemeComponentViewStyle> {
 
     override fun map(from: CardSchemeComponentStyle) = CardSchemeComponentViewStyle(
+        mainAxisSpacing = from.imageContainerStyle.mainAxisSpacing,
+        crossAxisSpacing = from.imageContainerStyle.crossAxisSpacing,
         titleStyle = from.titleStyle.let { textLabelStyleMapper.map(it) },
-        containerModifier = containerMapper.map(from.imagesContainerStyle).fillMaxWidth()
+        containerModifier = containerMapper.map(from.containerStyle).fillMaxWidth(),
+        imagesContainerModifier = containerMapper.map(from.imageContainerStyle.containerStyle).fillMaxWidth()
```

Comment:
It would be nice to create `FlowRowViewStyle`, as it can be useful in the future and will increase readability and responsibility separation in `CardSchemeComponentViewStyle`.
`FlowRowViewStyle` could contain `mainAxisSpacing, crossAxisSpacing, containerModifier`.